### PR TITLE
Sort sites menu

### PR DESCRIPTION
**Pull request by robsonsobral**
## Overview

I'm just sorting menu of sites. I don't think a single line deserves even a changelog.

Resolves [#NN](https://github.com/ExpressionEngine/ExpressionEngine/issues/NN).

## Nature of This Change

- [x] 🐛 Fixes a bug
- [ ] 🚀 Implements a new feature
- [ ] 🛁 Refactors existing code
- [ ] 💅 Fixes coding style
- [ ] ✅ Adds tests
- [ ] 👽 Adds new dependency
- [ ] 🔥 Removes unused files / code
- [ ] 🔒 Improves security 

## Is this backwards compatible?

- [x] Yes
- [ ] No

## Documentation

User Guide Pull Request: https://github.com/ExpressionEngine/ExpressionEngine-User-Guide/pull/NNN

### CODE REVIEW

File: system/ee/ExpressionEngine/View/_shared/sidebar/navigation/navigation.php [hunk 1]
```diff
@@ -10,6 +10,7 @@
 
 					<?php
                     if (!empty($sites)) :
+                    asort($sites);
```
Comment (intoeetive) [inline system/ee/ExpressionEngine/View/_shared/sidebar/navigation/navigation.php]: Could it be better to sort the sites when those are fetched? That would also give more freedom if we'd want to make sorting configurable (whether we want to have it by ID or name, etc)

**Comment by robsonsobral** [comment]
Do you want me to change it [here](https://github.com/ExpressionEngine/ExpressionEngine/blob/49d282f67c4ff5e7a79830bc8b94550b31013383/system/ee/legacy/libraries/Session.php#L1284)?

**Comment by intoeetive** [comment]
@robsonsobral I think that might be the best. It will make the list already sorted when it's being used elsewhere

**Comment by robsonsobral** [comment]
It's done, @intoeetive . However, I didn't like the result. I guess the previous code is safer.

**Comment by intoeetive** [comment]
Can you share why you think this code is not safe?

(also, would it be better if we sort before saving to `userdata`, not after?)

**Comment by robsonsobral** [comment]
I was in doubt because at some point, somebody chose to [reverse sort](https://github.com/ExpressionEngine/ExpressionEngine/blob/49d282f67c4ff5e7a79830bc8b94550b31013383/system/ee/legacy/libraries/Session.php#L1289) the sites and I didn't know why. So, I took sometime digging the file history and found it's a regression caused on a [commit](https://github.com/ExpressionEngine/ExpressionEngine/commit/203b551c98917782351408fb5330a2ac7bfca5e7#diff-dd0cee0b8a116d64bba79702feed27ea0c571adc834c3f4ae338a06774b4cf76R1373) by @obfuscode .

Can you, please, check if you agree to change the sorting order of the site here, @obfuscode ? (it's nothing personal!)

**Comment by obfuscode** [comment]
Do you have a link to a more specific commit @robsonsobral ? That commit link just goes to where Matt merged in 3,690 files? Nothing personal taken! I want to make sure we get the best version of the code in as well! :)

**Comment by obfuscode** [comment]
So I do see where in that link the change was made but not any reason why `desc` was specifically added.

**Comment by robsonsobral** [comment]
To me, it looks like a simple mistake when adapting the code to the PSR standard. If so, this fix can be rebased and merged.

(It's really weird to see the sites in reversed order. Why?!)

**Review by intoeetive — APPROVED** [review]
I have looked through the Git history and I was only able to track this down to merge commit by EllisLab back in 2019: https://github.com/EllisLab/ExpressionEngine-Private/commit/3b8aba2e586b4b5607b07760fe85507af706ec9d
It does not show specifically this change and it does not show the reasoning. I believe this was done my mistake.
Approving the PR

**Pull request merged**